ginga/gtk3w/Widgets.py

```diff
@@ -196,6 +216,12 @@ def set_enabled(self, tf):
         super(TextEntrySet, self).set_enabled(tf)
         self.entry.set_sensitive(tf)
 
+
+class GrowingTextEdit(WidgetBase):
+    def __init__(self, *args, **kwargs):
+        raise NotImplementedError
+
```

Comment:
I think `GrowingTextEdit` was a hack a I had to do for Qt and wasn't needed for Gtk because of the way the widget works.  I don't think it was meant to be instantiated directly by users.  But I should review that--it was far enough back that my memory is fuzzy on it,